torch/_inductor/codegen/multi_kernel.py

```diff
@@ -364,7 +374,7 @@ def inner():
             for index, kernel in enumerate(self.kernels)
         ]
 
-    def _get_filtered_args(self, args, index):
+    def _get_filtered_args(self, args: tuple[Any, ...], index: int) -> Union[list[Any], tuple[Any, ...]]:
```

Comment:
It could be a tuple of a type var matching the inputs, right?

```suggestion
    def _get_filtered_args(self, args: tuple[_T, ...], index: int) -> Union[list[Any], tuple[_T, ...]]: